src/main/java/appeng/core/lib/module/ModuleLoaderHelper.java

```diff
@@ -0,0 +1,98 @@
+package appeng.core.lib.module;
+
+import com.google.common.collect.Lists;
+import net.minecraftforge.fml.common.Loader;
+import net.minecraftforge.fml.relauncher.Side;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by Johannes on 13.09.2016.
+ */
+public class ModuleLoaderHelper {
```

Comment:
There was a reason why i kept it in pre init: we should load as few classes as possible. If we accidentaly load client module on server...